os-autoinst-obs-auto-submit

```diff
@@ -126,32 +126,33 @@ TMPDIR=${TMPDIR:-$(mktemp -d)}
 trap 'rm -rf "$TMPDIR"' EXIT
 
 (cd "$TMPDIR"
-for package in os-autoinst openQA os-autoinst-distri-opensuse-deps; do
-    $osc service wait "$src_project"/$package
-    $osc co -S "$src_project"/$package
+auto_submit_packages="$(osc ls "$dst_project" | grep -v '\-test$')"
+for package in $auto_submit_packages; do
+    $osc service wait "$src_project"/"$package"
+    $osc co -S "$src_project"/"$package"
     if test "$package" == "os-autoinst-distri-opensuse-deps"; then
         $osc cat "$submit_target" "$package" "$package.spec" > "$package-factory.spec"
-        if diff -u $package-factory.spec "$src_project"/$package/_service:obs_scm:$package.spec | grep "^[+-]Requires"; then
+        if diff -u "$package"-factory.spec "$src_project"/"$package"/_service:obs_scm:"$package".spec | grep "^[+-]Requires"; then
```

Comment:
It would likely look less noisy when using just one pair of quotes, e.g.
```suggestion
        if diff -u "$package"-factory.spec "$src_project/$package/_service:obs_scm:$package.spec" | grep "^[+-]Requires"; then
```

Reply:
That we can do